### PR TITLE
Small, but important changes

### PR DESCRIPTION
**Pull request by NazarYablonskiy**
- Removed
```
    <uses-permission android:name="android.permission.RECORD_AUDIO" />
    <uses-permission android:name="android.permission.RECORD_VIDEO" />
    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
```

- Updated example
- Updated README.md

##### Explanation
My company uses this module only for scanning of barcodes. Some users blame us in espionage (at least, I think so (according to user feedback)) because the app requires an access to a microphone.

### CODE REVIEW

**Comment by ptomasroos** [comment]
This is a quite breaking change. Personally I don't have a problem with changing this behaviour. 
But I'll let someone more provide feedback

**Comment by chris-at-translate** [comment]
In theory remove all permissions and then ask for the permissions when they are needed is the better way to go.

**Comment by ptomasroos** [comment]
@chris-at-translate that only works for devices with runtime permissions.

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "type": "git",
     "url": "https://github.com/lwansbrough/react-native-camera.git"
   },
-  "version": "0.6.0",
+  "version": "0.6.1",
```
Comment (rt2zz) [inline package.json]: remove this change, we will cut a new release separately

**Comment by rt2zz** [comment]
I like these changes, and agreed while it is a fundamentally breaking change, now is the time to make it. Can you add a section near the top of the docs explaining the change and why we made it.

**Comment by NazarYablonskiy** [comment]
@rt2zz done

**Comment by ptomasroos** [comment]
LGTM!

**Pull request merged**